extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -131,6 +131,17 @@ oppia.factory('ItemSelectionInputValidationService', [
                         'please select only one answer choice.')
                     });
                   }
+                } else if (rule.type === 'IsProperSubsetOf') {
+                  handledAnswers[choiceIndex] = true;
+                  if (ruleInput.length < 2) {
+                    warningsList.push({
+                      type: WARNING_TYPES.ERROR,
+                      message: (
+                        'In answer group ' + (answerIndex + 1) + ', ' +
+                        'rule ' + (ruleIndex + 1) + ', ' +
+                        'Is proper subset, needs at least 2 options')
```

Comment:
In general, the above condition looks good @amey-kudari as we have discussed. Have you understood why we need to add this warning now?

Nit: Add full-stop in 'needs at least 2 options' warning.

Reply:
Yeah, I understood why it is not possible to have <2, Thanks for helping! 
I will add the full-stop and add relevant test cases for this now.

Reply:
Added the tests,